packages/redux-saga/rollup.config.js

```diff
@@ -171,6 +173,24 @@ export default [
       name: 'ReduxSagaEffects',
     },
   }),
+  createConfig({
```

Comment:
I would skip those entirely, I expect there are not many UMD users - we can always add those configs when it gets requested by somebody.